### DISCUSSION TITLE
How to do "prose prose-sm md:prose"

### DISCUSSION THREAD

**Discussion by danspratling** [tailwindlabs/tailwindcss#6103] [Help]
So I've run into this weird issue where I'm trying to have smaller prose content on mobile, and the default size on other screen sizes

Looking at the docs, `prose` is a required class regardless of screen size (it adds additional things that the size modfiers don't)

But because that class is required, it makes it impossible to refer to it as a size modifier when adding a media query. If I do `prose-sm md:prose` things start disappearing (like quotes) on mobile

If I wanted to use any other font size, i would use a prose size modifier

```
prose lg:prose-lg xl:prose-xl
```

so it seems like this missing base sizing is a mistake

## Suggestion

A new size modifier is added with base size styling to allow media query adjustments

```
prose prose-sm md:prose-base
// Where prose-base resets the font (and any other scaling/changes) to the default size
````

## workaround (edit, doesn't work)

for now, I can just manually overwrite the base font size by doing the following

```
prose prose-sm md:text-base
```

But if I've customized prose so that something not accounted for by rem, like colours, are different on sm and DEFAULT in my tailwind config, there's no way I can reset them to the default without manually adjusting all styles (but with all other prose types this blanket reset is possible)

Edit: After trying this workaround I realised that it only works on base paragraph styles. Headings (h2, h3, etc) still inherit the prose-sm styles which makes them smaller on larger screens than intended

**Comment by adamwathan** [comment]
Hey! You just want this:

```
prose prose-sm md:prose
```

In the next version we will have a `prose-base` modifier you can use as well.

**Reply by danspratling** [reply]
Hey Adam

Doesn't `prose` cancel out the `md:prose` in that situation?

prose (adds prose classes on every screen size)
md:prose (adds prose classes above md screen size)

Or am I missing something with how these are set up hierarchically? When I attempted it using `prose prose-sm md:prose` it didn't work as expected (it retained prose-sm sizing above the md breakpoint)

**Reply by Pfed-prog** [reply]
nope, prose seems to be necessary
